esmvaltool/_main.py

```diff
@@ -56,6 +57,17 @@
 """ + __doc__
 
 
+def remove_preproc(preproc_dir):
```

Comment:
Please use [`shutil.rmtree`](https://docs.python.org/3.6/library/shutil.html#shutil.rmtree) instead of writing your own function to remove a directory.